### PR TITLE
[docs] update to v1.9 API versions

### PR DESCRIPTION
**Pull request by DirectXMan12**
This updates the README and walkthrough to use v1.9 API versions,
and to use Prometheus v2.

### CODE REVIEW

**Comment by DirectXMan12** [comment]
@brancz @luxas can you guys do a quick sanity check here?

File: docs/walkthrough.md [hunk 1]
```diff
@@ -156,15 +160,13 @@ spec:
       labels:
         app: prometheus
     spec:
-      serviceAccountName: prom-cm-adapter 
+      serviceAccountName: prom-cm-adapter
       containers:
-      - image: prom/prometheus:v1.6.1
+      - image: prom/prometheus:v2.1.0
```
Comment (brancz) [inline docs/walkthrough.md]: there are some known problems with v2.1.0, v2.2.0-rc.0 is probably more stable

File: docs/walkthrough.md [hunk 2]
```diff
@@ -310,19 +312,22 @@ Take the resulting value, and place it into the following file:
 
 <summary>cm-registration.yaml</summary>
 
+*Note that apiregistration moved to stable in 1.10, so you'll need to use
+the `apiregistration.k8s.io/v1` API version there*.
```
Comment (brancz) [inline docs/walkthrough.md]: I was not aware of this, neat!

File: docs/walkthrough.md (hunk 1 above)
Comment (DirectXMan12) [reply docs/walkthrough.md]: ack

**Comment by brancz** [comment]
lgtm

File: docs/walkthrough.md (hunk 2 above)
Comment (DirectXMan12) [reply docs/walkthrough.md]: https://github.com/kubernetes/kubernetes/pull/58393

**Comment by DirectXMan12** [comment]
thanks for the review.  Comments addressed.  Will merge shortly

**Pull request merged**